### ISSUE TITLE
linker error building xrdp-droppriv

### ISSUE THREAD

**Issue by jsorg71** [neutrinolabs/xrdp#3160]
### xrdp version

devel

### Detailed xrdp version, build options

~~~
./bootstrap
./configure
make
~~~

### Operating system & version

devuan 4(debian 11)

### Installation method

git clone & make install

### Which backend do you use?

Xorg

### What desktop environment do you use?

KDE

### What's your client?

MSTSC

### Area(s) with issue?

Other

### Steps to reproduce

just build on this platform

### ✔️ Expected Behavior

linking should complete

### ❌ Actual Behavior

build failure

### Anything else?

```
make[3]: Entering directory '/home/jay/git/neutrinolabs/xrdp/tools/chkpriv'
  CC       xrdp-droppriv.o
  CCLD     xrdp-droppriv
/usr/bin/ld: xrdp-droppriv.o: undefined reference to symbol 'dlclose@@GLIBC_2.2.5'
/usr/bin/ld: /lib/x86_64-linux-gnu/libdl.so.2: error adding symbols: DSO missing from command line
```

**Comment by jsorg71** [comment]
here is a patch that fixes it.  If it looks good I'll do a PR.
```
diff --git a/tools/chkpriv/Makefile.am b/tools/chkpriv/Makefile.am
index c6b1d04b..5c0713a6 100644
--- a/tools/chkpriv/Makefile.am
+++ b/tools/chkpriv/Makefile.am
@@ -14,7 +14,8 @@ AM_CPPFLAGS = \
 xrdp_droppriv_SOURCES = xrdp-droppriv.c
 
 xrdp_droppriv_LDADD = \
-  $(top_builddir)/common/libcommon.la
+  $(top_builddir)/common/libcommon.la \
+  $(DLOPEN_LIBS)
 
 SUBST_VARS = sed \
    -e 's|@pkglibexecdir[@]|$(pkglibexecdir)|g'

```

**Comment by matt335672** [comment]
LGTM - thanks.

**Comment by jsorg71** [comment]
#3161 for this

**Comment by metalefty** [comment]
Fixed by #3161
